src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --primary-color: #00acdc;
+  --font-family: font-family: Roboto, Arial, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+a,
+h1,
+h2,
+h3,
+p,
+ul {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+}
```

Comment:
Using tag names for styling is not a good practice as it can lead to unexpected results when the HTML structure changes. It's better to use class selectors. Also, the `text-decoration: none;` property is not applicable to `h1`, `h2`, `h3`, `p`, `ul` elements.